lib/components/molecules/ArticleThumbnail/ArticleThumbnail.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+/**
+ *
+ * ArticleThumbnail
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './ArticleThumbnail.style';
+import type { Props } from './types';
+import Image from '../../atoms/Image/Image';
+import Heading from '../../atoms/Heading/Heading';
+import Anchor from '../../atoms/Anchor';
+import Para from '../../atoms/Para';
+
+const getAnchoredChild = (url, children, target) => (
+  <Anchor to={url} target={target}>
+    {children}
+  </Anchor>
+);
+
+const getBaseVariation = (image, title, url, description, variation, readMoreText) => (
+  <React.Fragment>
+    {variation === 'v3' ? getAnchoredChild(url, <Image {...image} />) : <Image {...image} />}
+    {variation === 'v3' ? (
+      getAnchoredChild(url, <Heading>{title}</Heading>)
+    ) : (
+      <Heading>{title}</Heading>
+    )}
+
+    <Para>{description}</Para>
+    {variation === 'v2' ? getAnchoredChild(url, readMoreText) : ''}
```

Comment:
Again drive it with data, if props get config or data for read more you render it else not. :)

Reply:
Done